pkgs/tools/graphics/gmic-qt/default.nix

```
@@ -97,7 +99,17 @@ mkDerivation rec {
     wrapQtApp "$out/${gimp.targetPluginDir}/gmic_gimp_qt/gmic_gimp_qt"
   '';
 
+  passthru = {
+    tests = {
+      gimp-plugin = gimpPlugins.gmic-qt;
```

```suggestion
      gimp-plugin = gimpPlugins.gmic;
```

I'm pretty sure it's just `gimpPlugins.gmic`, right?

---

Good catch.